translations/all_map_locations.tex/he.po

```diff
@@ -85,6 +87,30 @@ msgid ""
 "\\endgroup\n"
 "\n"
 msgstr ""
+"\\begingroup\n"
+"  \\renewcommand{\\arraystretch}{1.5}\n"
+"  \\newcolumntype{Y}{>{\\centering\\arraybackslash}X}\n"
+"  \\begin{tabularx}{\\linewidth}{Y m{0.8\\linewidth}}\n"
```

Comment:
```suggestion
"  \\begin{tabularx}{\\linewidth}{Y m{0.2\\linewidth}}\n"
```
This will help the swap.

Reply:
<img width="408" alt="image" src="https://github.com/user-attachments/assets/921e8f77-51d0-4af4-ad8f-d39fc1547083">

looking better!
Im trying to play with the location of the treasure_die to see if it will be big again